.github/workflows/cicd_mainnet_prod.yaml

```diff
@@ -348,6 +359,7 @@ jobs:
           ENABLE_PROCESS_BLOCK_RANGE_avalanche: 'true'
           ENABLE_PROCESS_BLOCK_RANGE_ethereum: 'true'
           ENABLE_PROCESS_BLOCK_RANGE_polygon: 'true'
+          ENABLE_PROCESS_BLOCK_RANGE_base: 'true'
           #
           AVALANCHE_NETWORK: 'avax'
```

Comment:
@wehappyfew I don't know the implications of these names but we might eventually want to update them to be the correct network names (`ethereum`, `avalanche`, `polygon`)

Reply:
@alexanderattar the network names' values [eg either 'avax' or avalanche'] need to be supported by your code. If that is the case, and nothing breaks, I can change them :]